components/email/src/Email.svelte

```diff
@@ -159,6 +189,36 @@
     );
   }
 
+  // #region get contact for ContactImage
+  let contactQuery: ContactsQuery;
+  $: contactQuery = {
+    component_id: id,
+    access_token,
+  };
+
+  /*
+    Fetches contact for ContactImage component
+  */
+  async function getContact(account) {
```

Comment:
Might be for another PR, but: I think we need to move this query method into a store and utilize the store for caching. Otherwise, we're going to be making multiple requests for the same email lookup multiple times.